ReadMe.md

```diff
@@ -30,57 +25,96 @@ The Aiko Engine MP provides ...
+- Download the Aiko software code from https://github.com/geekscape/aiko_engine_mp
 
-- Download the source code from <https://github.com/geekscape/aiko_engine_mp>
 ```
     git clone https://github.com/geekscape/aiko_engine_mp
+```
+**Note**: If you don't have Git installed you can download a zip file from https://github.com/geekscape/aiko_engine_mp and unpack it
+
+
+```
     cd aiko_engine_mp
 ```
-- Ensure you have a
-  [compatible hardware board](https://github.com/micropython/micropython/wiki/Boards-Summary) or
-  [compatible microPython port](https://github.com/micropython/micropython/tree/master/ports)
-- Make sure you have a Python
-  [virtual environment](http://docs.python-guide.org/en/latest/dev/virtualenvs/#lower-level-virtualenv) set-up, including
-  [virtualenvwrapper](http://docs.python-guide.org/en/latest/dev/virtualenvs/#virtualenvwrapper)
-- Create `mkvirtualenv micropython` and work on the new virtual environment
+
+**Note**: All commands below assume that `.../aiko_engine_mp` is your current directory.
+
+
+- Setup a [virtual Python environment](https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/#creating-a-virtual-environment) for your badge project. e.g.
+
 ```
-    workon micropython
+python3 -m venv .venv
 ```
-- Install mpfshell
+
+- Make sure your Python virtual environment is active. **Note**: Do this every time a new terminal session is started
+
+```
+. .venv/bin/activate # Linux and macOS
+. .venv\Scripts\activate.ps1 # Windows
+```
+
+- Install Python packages [`esptool`](https://github.com/espressif/esptool/blob/master/README.md) and [`mpfshell`](https://github.com/wendlers/mpfshell/blob/master/README.md)
+
 ```
-    pip install esptool
-    pip install -U mpfshell
+pip install -r requirements.txt
 ```
-- Plug in your ESP32 device and make sure you can see it,
-  e.g `ls /dev/tty.*` provides e.g `/dev/tty.wchserial1410`
-- Export the serial port to an environment variable, so mpfshell can use it,
-  e.g `export AMPY_PORT=<port>` where `port` is the device address shown
-  by the `ls` command above
-- Download latest [microPython](http://micropython.org/download)
-- Flash microPython
+
+### 2. Download and install MicroPython
+
+- Download the latest MicroPython binary for your ESP32 board from https://micropython.org/download/
+
+  Note that you must have a
+    [compatible hardware board](https://github.com/micropython/micropython/wiki/Boards-Summary) or
+    [compatible microPython port](https://github.com/micropython/micropython/tree/master/ports)
+
+- Plug in your ESP32 device and make sure you can see it. For example on
+  - macOS or Linux:
+  `ls /dev/tty.*` provides something similar to `/dev/tty.wchserial1410`
+  - On Windows, use the command `change port /query` to discover the COM port.
+
+- Export the serial port to an environment variable, so helpder sripts can use it,
+  e.g `export AMPY_PORT=<port>` or  `$env:AMPY_PORT = COM<n>` where `port` is the device address shown
```

Comment:
Perhaps note they can get straight to the badge REPL then with `mpfshell -o ${AMPY_PORT:4}`, at least with a POSIXish shell?